### PR TITLE
 Add possibility to add RestartlessMenuItems to Appmenu taskPopup menu 

### PR DESCRIPTION
**Pull request by Brummolix**
(the hamburger menu at the right side)

### CODE REVIEW

**Comment by Standard8** [comment]
@Brummolix Please note, I've just landed a patch to enable ESLint on this repository, which enforces rules about style, and potential issues in this code.

Please ensure `npm install` && `npm run test` passes when updating.

File: RestartlessMenuItems.js [hunk 1]
```diff
@@ -75,13 +76,16 @@ var global = this;
  */
 function monkeyPatchWindow(w, loadedAlready, options) {
   let doIt = function () {
-    let id = options.id;
-    let taskPopup = w.document.getElementById("taskPopup");
-    let tabmail = w.document.getElementById("tabmail");
-    let oldMenuitem = w.document.getElementById(id);
+	let id = options.id;
```
[the remark this answers is not in the document]
Comment (Brummolix) [reply RestartlessMenuItems.js]: Changed the spacings and npm check runs sucesfull

File: RestartlessMenuItems.js [hunk 2]
```diff
@@ -96,6 +100,20 @@ function monkeyPatchWindow(w, loadedAlready, options) {
       openTabUrl() || options.onCommand && options.onCommand();
     };
 
+	if (id && taskPopup)
+		addMenuItem(w,onCmd,options,id,taskPopup);
```
[the remark this answers is not in the document]
Comment (Brummolix) [reply RestartlessMenuItems.js]: Changed the spacings and npm check runs sucesfull

File: RestartlessMenuItems.js [hunk 3]
```diff
@@ -96,6 +100,20 @@ function monkeyPatchWindow(w, loadedAlready, options) {
...
+		addMenuItem(w,onCmd,options,idAppMenu,appMenuPopup);
+  };
+  if (loadedAlready)
+    doIt();
+  else
+    w.addEventListener("load", doIt, false);
+}
+
+function addMenuItem(w,onCmd,options,id,taskPopup)
+{
```
[the remark this answers is not in the document]
Comment (Brummolix) [reply RestartlessMenuItems.js]: Changed the spacings and npm check runs sucesfull

File: RestartlessMenuItems.js [hunk 4]
```diff
@@ -75,13 +76,16 @@ var global = this;
...
+	let idAppMenu = options.idAppMenu;
+
+	let taskPopup = w.document.getElementById("taskPopup");
+	let appMenuPopup = w.document.getElementById("appmenu_taskPopup");
 
+    let tabmail = w.document.getElementById("tabmail");
+	
     // Check the windows is a mail:3pane
-    if (!taskPopup || !tabmail)
+    if ( (!taskPopup && !appMenuPopup) || !tabmail)
```
[the remark this answers is not in the document]
Comment (Brummolix) [reply RestartlessMenuItems.js]: Changed the spacing

**Comment by Brummolix** [comment]
I tried to solve all review issues.
Additionally I had some problems on Windows. Because my git setting normally changes all line endings to CRLF the eslint checks were failing. Therefore I additionally added a .gitattribute to force LF endings.

**Comment by Standard8** [comment]
I'm sorry for never having got around to merging this. Giving Thunderbird's moving away from the legacy style extensions, and heading towards WebExtensions, I've decided to remove this file as Conversations no longer needs it, and I'm not intending on supporting it in future.

**Pull request closed without merging**